mobile/src/reducers/feeds/feed.model.ts

```diff
@@ -0,0 +1,24 @@
+import {dataMember, required} from 'santee-dcts';
+
+export class Feed {
+
+    @dataMember()
+    @required()
+    public messageId: string;
+
+    @dataMember()
+    @required()
+    public employeeId: string;
+
+    @dataMember()
+    @required()
+    public title: string;
+
+    @dataMember()
+    @required()
+    public text: string;
+
+    @dataMember()
+    @required()
+    public datePosted: string;
```

Comment:
there is something like customDeserializer parameter to dataMember, which you can use to convert this string directly to moment object